tpls/mdspan/include/experimental/__p0009_bits/config.hpp

```diff
@@ -116,7 +116,7 @@ static_assert(MDSPAN_IMPL_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14
 
 #if !defined(MDSPAN_IMPL_USE_ATTRIBUTE_NO_UNIQUE_ADDRESS)
 #  if ((MDSPAN_IMPL_HAS_CPP_ATTRIBUTE(no_unique_address) >= 201803L) && \
-       (!defined(__NVCC__) || (__CUDACC_VER_MAJOR__ < 13 && MDSPAN_HAS_CXX_20)) && \
+       (!defined(__NVCC__) || ((__CUDACC_VER_MAJOR__ * 100 + __CUDACC_VER_MINOR__ * 10 < 1209) && MDSPAN_HAS_CXX_20)) && \
```

Comment:
Shouldn't that be
```suggestion
       (!defined(__NVCC__) || ((__CUDACC_VER_MAJOR__ * 100 + __CUDACC_VER_MINOR__ * 10 < 1290) && MDSPAN_HAS_CXX_20)) && \
```
(and similar elsewhere)?

Reply:
Why not?
```
(__CUDACC_VER_MAJOR__ >= 12) && (__CUDACC_VER_MINOR__ >= 9)
```
I think that is easier to read

Reply:
Either way this needs to be done in `mdspan`.

Reply:
> Why not?
> 
> ```
> (__CUDACC_VER_MAJOR__ >= 12) && (__CUDACC_VER_MINOR__ >= 9)
> ```
> 
> I think that is easier to read

I think if we go that way we need `(__CUDACC_VER_MAJOR__ >= 13) || ((__CUDACC_VER_MAJOR__ == 12) && (__CUDACC_VER_MINOR__ >= 9))`. Do you find it much simpler ?